yogstation/code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -0,0 +1,946 @@
+#define EMPOWERED_THRALL_LIMIT 5
+
+/obj/effect/proc_holder/spell/proc/shadowling_check(var/mob/living/carbon/human/H)
+	if(!H || !istype(H)) return
+	if(H.dna.species.id == "shadowling" && is_shadow(H)) return TRUE
+	if(H.dna.species.id == "l_shadowling" && is_thrall(H)) return TRUE
```

Comment:
I'm pretty sure you need to check if ``dna`` and ``dna.species`` is valid. @nichlas0010